tekton/src/pipelines/fvt-optimizer.yml.j2

```diff
@@ -509,15 +545,18 @@ spec:
         - input: "$(params.fvt_digest_ctf)"
           operator: notin
           values: [""]
+        - input: "$(params.mas_app_channel_optimizer)"
+          operator: in
+          values: ["9.0.x-dev", "9.0.x", "8.5.x-dev", "8.5.x", "8.4.x-dev", "8.4.x"]
```

Comment:
@andrefgj in our ai-solution containers, as we are child of ubi-fvt, we should be able also to use the after/before markers on top of the functions... to keep it consistent wit hthe rest of the testw in FVT, can we do that way? The more we avoid rules like these in `when` clause the better so we can keep the FVT report consistent (and not have some holes in results, caused by not running the task in a pipeline level)